### ISSUE TITLE
Dependency conflicts when using with Aqueduct 3.0 (Dart2)

### ISSUE THREAD

**Issue by aspantel** [PointyCastle/pointycastle#144]
We're migrating from Aqueduct 2.x (Dart1) to 3.0 (Dart2) and now are stuck with the conflict.
Would you be able to upgrade your package's dependencies or suggest a workaround?

Thank you.

Because reflectable >=2.0.2 depends on analyzer ^0.31.0 and reflectable >=0.1.1 <2.0.2 requires SDK version >=1.8.0 <2.0.0-dev.infinity, reflectable >=0.1.1 requires analyzer ^0.31.0.
And because aqueduct >=3.0.0-beta.1 depends on analyzer ^0.32.0 and pointycastle >=1.0.0-rc1 depends on reflectable >=1.0.1 <3.0.0, aqueduct >=3.0.0-beta.1 is incompatible with pointycastle >=1.0.0-rc1.
So, because threadbot depends on both pointycastle ^1.0.0-rc2 and aqueduct ^3.0.0, version solving failed.

**Comment by aspantel** [comment]
reflectable >=2.0.0 depends on build_barback >=0.0.2 which requires SDK version >=1.9.1 <2.0.0 or >=2.0.0-dev.9 <2.0.0, reflectable >=2.0.0 is forbidden

**Comment by yshrsmz** [comment]
possible duplicate of #119 ?
and it's blocked by #131
